src/main.js

```diff
@@ -2,11 +2,17 @@ import {createSiteMenuTemplate} from "./view/site-menu.js";
 import {createSiteFilterTemplate} from "./view/site-filter.js";
 import {createBoardElement} from "./view/board-element.js";
 import {createBoardFiltersTemplate} from "./view/board-filters.js";
-import {editCardTemplate} from "./view/edit-card";
-import {createCardTemplate} from "./view/create-card";
-import {createloadButtonTemplate} from "./view/load-button";
+import {editCardTemplate} from "./view/edit-card.js";
+import {createCardTemplate} from "./view/create-card.js";
+import {createloadButtonTemplate} from "./view/load-button.js";
```

Comment:
Название функций должны отражать действие (https://up.htmlacademy.ru/ecmascript/12/criteries#b10)
```suggestion
import {getEditCardTemplate} from "./view/edit-card.js";
import {getCreateCardTemplate} from "./view/create-card.js";
import {getLoadButtonTemplate} from "./view/load-button.js";
```